src/ui/common/src/components/pages/LoginPage.tsx

```diff
@@ -14,8 +14,13 @@ export const LoginPage: React.FC = () => {
   const [apiKey, setApiKey] = useState<string>(
     cookies.load('aqueduct-api-key')
   );
+
   const navigate = useNavigate();
 
+  const [isAuthed, setIsAuthed] = useState<boolean>(false);
+  // used to trigger a re-render of the page after auth has gone through.
+  const [toggle, setToggle] = useState<boolean>(false);
```

Comment:
Probs don't need this anymore. Can't quite figure out why this works only one time around and why removing cookies and trying again doesn't work.

Removing the cookie does bounce me back here though, which is good.